config/locales/es.yml

```diff
@@ -2315,6 +2315,7 @@ es:
       data_transfer_offboarding:
         edit:
           ineligible_reason:
+            exempt_interest_exceeds_10k: your tax exempt interest income exceeds $10,000.
```

Comment:
[pebble] does this need a translation?

Reply:
Yes, we're going to make English and Spanish content updates in bulk later once the text is finalized. We're tracking bad placeholder text in a ticket to make sure we catch it